vlib/v/checker/tests/optional_variable_err.vv

```diff
@@ -0,0 +1,4 @@
+fn main() {
+	x := !bool(false)
+	println(x)
+}
```

Comment:
this is better done as a _test.v file inside vlib/v/tests/

Reply:
Oh sorry, it should be ?bool(false)